### DISCUSSION TITLE
Welcome to Canarytokens Docker Discussions!

### DISCUSSION THREAD

**Discussion by jayjb** [thinkst/canarytokens-docker#145] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lephuhung** [comment]
Can I turn off CORS or allow: * to load url token from img src
